e2e-polybft/e2e/storage_test.go

```diff
@@ -72,7 +72,6 @@ func TestE2E_Storage(t *testing.T) {
 			txn.SetNonce(uint64(i))
 
 			tx := cluster.SendTxn(t, sender, txn)
-			err = tx.Wait()
```

Comment:
Not sure if it is ok to remove this line, this way we don't know the result of tx processing. This is the place where we will get the result of processing and next require.NoError should check if processing succeeded.

Reply:
No need to wait. The wait is done in the `SendTxn` function which uses the `txRelayer` package.

Reply:
Ok, thanks. Then following line with require.NoError should be replaced with different type of check.

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/174/commits/60a82271b322ba4ddb2f2c38aa9b2a17b925d862